### PR TITLE
Added django 2.2 support

### PR DESCRIPTION
**Pull request by lino**
tox tests are passing

### CODE REVIEW

**Comment by zsoldosp** [comment]
Hi @lino, thanks for the PR, it looks good to me! :+1:

When new releases are made, the following changes also should happen

* bump the version in `django_currentuser/__init__.py` - in this case, only the build number (last) needs to increase, as no API changes or features were introduced/removed
* update `.travis.yml` - just running `tox2travis.py` should take care of that (or maybe need to pipe the output into `.travis.yml`)
* update the `README.md` with release notes - however, as I see it has been, ahem, a bit behind so we should catch up on that

However, I recall that for releases there was a problem with 3.7 python recently, cc'ing @kairichard and @Eorhim for input on that

File: tox.ini [hunk 1]
```diff
@@ -4,6 +4,7 @@ envlist =
    py{27,34,35,36}-django111,
    py{34,35,36}-django20,
```
Comment (Eorhim) [inline tox.ini]: Python 3.4 reached end-of-life. No need to test for it anymore.

File: tox2travis.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ def matrix(self):
             py34='3.4',
```
Comment (Eorhim) [inline tox2travis.py]: End-of-life for Python 3.4

**Comment by Eorhim** [comment]
@lino, @zsoldosp 

The problem with Python 3.7 was solved in [fe951cf6807beb9671734a2a13ffb07391fa7a81](https://github.com/PaesslerAG/django-currentuser/commit/fe951cf6807beb9671734a2a13ffb07391fa7a81) by cutting out some outdated build system and using the Travis functionality directly. 

Unfortunately `tox2travis.py` wasn't updated. So if you want to use that script to generate the `.travis.yml` configuration, you will loose the packaging ability.

**Comment by Eorhim** [comment]
@lino: I close this pull request, because I extended your work in pull request #29.

Thank you for pushing the project forward with your work.

**Pull request closed without merging**